Plugins/Alpakit/Source/Alpakit/Private/AlpakitSettings.cpp

```diff
@@ -1,9 +1,51 @@
 #include "AlpakitSettings.h"
 
+#include "Json.h"
+
 UAlpakitSettings* UAlpakitSettings::Get() {
-    return GetMutableDefault<UAlpakitSettings>();
+	return GetMutableDefault<UAlpakitSettings>();
 }
 
 void UAlpakitSettings::SaveSettings() {
-    SaveConfig();
+	SaveConfig();
+}
+
+
+UAlpakitModSelectionSettings* UAlpakitModSelectionSettings::Get() {
+	return GetMutableDefault<UAlpakitModSelectionSettings>();
+}
+
+void UAlpakitModSelectionSettings::SaveSettings() {
+	SaveConfig();
+}
+
+void UAlpakitModSelectionSettings::loadJson() {
```

Comment:
Just... why? Why did you need all of that stuff? Do you know TMap<FString, bool> can be marked as properties too and saved/persisted? What's your motivation behind that code?

Reply:
Using tmap caused UE crash 🤷🏽‍♂️